### DISCUSSION TITLE
How do I use coc diagnostics?

### DISCUSSION THREAD

**Discussion by vpfeiffer** [neoclide/coc.nvim#4169] [Q&A] [answered]
Does anyone here use coc with neovim and if so do you know how to bring up the diagnostics list? I tried in neovim using:CocDiagnostics but it just brought up a small blank window.

**Comment by fannheyward** [comment]
Blank list means no diagnostics found. Needs more info to debug what's wrong.

**Reply by vpfeiffer** [reply]
Thanks!

**Comment by chemzqm** [comment, marked as answer]
It's location list for diagnostics of current buffer

**Reply by vpfeiffer** [reply]
Thanks!
